test/controllers/lists_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class ListsControllerTest < ActionDispatch::IntegrationTest
+
+  test "should get index" do
+    get lists_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_list_url
+    assert_response :success
+  end
+
+  test "should create list" do
+    assert_difference('List.count') do
+      post lists_url, params: { list: { title: @list.title, description: @list.description } }
```

Comment:
these look really good. Nice to see that you where able to get up to speed on minitest